doc/images/plugins.dot

```diff
@@ -150,7 +150,6 @@ strict digraph G {
 	process [shape=Mdiamond];
 	profile [shape=Mdiamond];
 	python [shape=Mdiamond];
-	python2 [shape=Mdiamond];
```

Comment:
This is removed when running `scripts/dev/draw-all-plugins`, which also fixes the png automatically.

Reply:
there is a png? 🤯